### PR TITLE
LGA-1538 erratic drop-down fields on contact page

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
## What does this pull request do?

Defines a class attribute for all drop-downs.  

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/templates/macros/subform.html [hunk 1]
```diff
@@ -42,7 +42,9 @@
       <li>{{ _('If your nominated person misses the call you’ll need to complete this form again.') }}</li>
     </ul>
     {{ Form.group(form.thirdparty.full_name, field_attrs={'class': 'm-large'}) }}
-    {{ Form.group(form.thirdparty.relationship) }}
+    {{ Form.group(form.thirdparty.relationship,
+      '',
```
Comment (exonian) [inline cla_public/templates/macros/subform.html:46]: What is this empty argument? I can see it in many but not all (eg `form.thirdparty.contact_number` directly below) instances

Comment (MalcolmVonMoJ) [reply cla_public/templates/macros/subform.html:46]: Not 100% sure.  I just copied the pattern, including the empty argument.  I could see element classes were passed into it in some instances.  

Looking into if further, this passes extra classes into the group element, such as the conditional element for show-hide things.  

I am surprised to see it declared null, like you obviously are, but I just accepted it and carried on.

Comment (MalcolmVonMoJ) [reply cla_public/templates/macros/subform.html:46]: I've just tried out removing it and it seems to work fine.  I can't see any difference in the rendered HTML.  I shall remove the remaining ones.

File: cla_public/templates/macros/subform.html [hunk 2]
```diff
@@ -42,7 +42,8 @@
       <li>{{ _('If your nominated person misses the call you’ll need to complete this form again.') }}</li>
     </ul>
     {{ Form.group(form.thirdparty.full_name, field_attrs={'class': 'm-large'}) }}
-    {{ Form.group(form.thirdparty.relationship) }}
+    {{ Form.group(form.thirdparty.relationship,
+      field_attrs={'class': ''}) }}
```
Comment (mayowaadeniyi) [inline cla_public/templates/macros/subform.html:46]: Can we remove the linespace so it's on one line? Just so it's a bit more readable 😅

**Review by mayowaadeniyi — APPROVED** [review]
LGTM 👍

**Pull request merged**